x-pack/plugins/security_solution/public/detection_engine/rule_creation_ui/components/description_step/helpers.tsx

```diff
@@ -69,6 +69,27 @@ const Query = styled.div`
   white-space: pre-wrap;
 `;
 
+export const getQueryLabel = (ruleType: Type | undefined): string => {
+  switch (ruleType) {
+    case 'query':
+      return i18n.QUERY_LABEL;
+    case 'eql':
+      return i18n.EQL_QUERY_LABEL;
+    case 'saved_query':
+      return i18n.SAVED_QUERY_LABEL;
+    case 'threshold':
+      return i18n.QUERY_LABEL;
+    case 'threat_match':
+      return i18n.THREAT_QUERY_LABEL;
+    case 'new_terms':
+      return i18n.QUERY_LABEL;
+    case 'esql':
+      return i18n.ESQL_QUERY_LABEL;
+    default:
+      return i18n.QUERY_LABEL;
+  }
```

Comment:
```suggestion
  switch (ruleType) {
    case 'eql':
      return i18n.EQL_QUERY_LABEL;
    case 'saved_query':
      return i18n.SAVED_QUERY_LABEL;
    case 'esql':
      return i18n.ESQL_QUERY_LABEL;
    case 'query':
    case 'threshold':
    case 'threat_match':
    case 'new_terms':
    default:
      return i18n.QUERY_LABEL;
  }
```

multiple choices can be combined like this.

I would even suggest to consider removing custom query options in favour of default like:

```suggestion
  switch (ruleType) {
    case 'eql':
      return i18n.EQL_QUERY_LABEL;
    case 'saved_query':
      return i18n.SAVED_QUERY_LABEL;
    case 'esql':
      return i18n.ESQL_QUERY_LABEL;
    default:
      return i18n.QUERY_LABEL;
  }
```